src/wallet/rpc/coins.cpp

```diff
@@ -676,7 +676,7 @@ RPCHelpMan listunspent()
                 entry.pushKV("ancestorfees", uint64_t(ancestor_fees));
             }
         }
-        entry.pushKV("spendable", out.spendable);
+        entry.pushKV("spendable", true); // Any coins we list are always spendable
```

Comment:
The doc needs to be updated as well. This would close https://github.com/bitcoin/bitcoin/issues/33110.

```diff
- {RPCResult::Type::BOOL, "spendable", "Whether we have the private keys to spend this output"},
+ {RPCResult::Type::BOOL, "spendable", "deprecated (always true)"},
```

Reply:
nit (not blocker, only if you have to retouch or could be done on a follow-up):
```diff
- {RPCResult::Type::BOOL, "spendable", "deprecated (always true)"},
+ {RPCResult::Type::BOOL, "spendable", "(DEPRECATED) Always true for UTXOs belonging to a descriptor wallet, including ones without private keys"},
```